src/components/Thread/Thread.vue

```diff
@@ -223,6 +223,14 @@ export default {
             { deep: true, immediate: true }
         );
 
+        this.$watch(
+            '$refs.sendbar.show_templates',
+            () => Vue.nextTick( () => { 
+                this.margin_bottom = this.$refs.sendbar && this.$refs.sendbar.$el.clientHeight ;
+            }),
+            { immediate: true }
+        );
+
```

Comment:
I think it needs to be watched because watching it will make it robust as we can add more things in future which can have variable height. You are right about `Scrolling to bottom`

Reply:
Adding a watcher is a significant amount of overhead for little benefit. As there is already one on this page, I would recommend not including it until its needed. 

Robustness, in this case, doesn't seem worth while when good enough does the job.

Reply:
Sure, I will make the change.